Lib/pdb.py

```diff
@@ -1388,8 +1388,13 @@ def do_break(self, arg, temporary=False):
             try:
                 lineno = int(arg)
             except ValueError:
-                self.error('Bad lineno: %s' % arg)
-                return
+                func = arg
+                find_res = find_function(func, self.canonic(filename))
+                if not find_res:
+                    self.error('Bad lineno or function name: %s' % arg)
+                    return
+                funcname, filename, ln = find_res
+                lineno = int(ln)
```

Comment:
I checked the define of `find_function`, and found that the third result is already an int, so the cast is not needed?

```suggestion
                funcname, filename, lineno = find_res
```

Reply:
Yes, because at first i used `Pdb().lineinfo()` instead of `find_function` here, so i need to do the cast.